.eslintrc.json

```diff
@@ -0,0 +1,58 @@
+{
+    "root": true,
+    "env": {
+        "browser": true,
+        "mocha": true,
+        "jquery": true,
+        "es6": true,
+        "node": true
+    },
+    "globals": {
+        "YT": true,
+        "ga": true,
+        "twttr": true,
+        "time": true,
+        "plyr": true
+    },
+    "rules": {
+        "indent": [
+            2,
+            4,
+            {
+                "SwitchCase": 1
+            }
+        ],
+        "curly": 2,
+        "linebreak-style": [
+            2,
+            "unix"
+        ],
+        "camelcase": [
+            2,
+            {
+                "properties": "always"
+            }
+        ],
+        "no-loop-func": 2,
+        "quotes": [
+            2,
+            "double",
+            "avoid-escape"
+        ],
+        "no-trailing-spaces": 2,
+        "no-unused-expressions": 2,
+        "no-console": 1,
+        "semi": [
+            2,
+            "always"
+        ],
+        "eqeqeq": 2,
+        "space-before-blocks": 2,
+        "spaced-comment": 2,
+        "space-infix-ops": 2,
+        "space-after-keywords": "off",
+	"keyword-spacing": [2, {"before": true, "after": true}],
```

Comment:
There's a tab on that line btw. I removed the line, and ran `node_modules/eslint/bin/eslint.js main.js -c .eslintrc.json`. There's failures because all the indents are off by 1 space, I fixed those and eslint passes